cmd/shell.go

```diff
@@ -28,6 +28,7 @@ import (
 	"fmt"
 	"io/ioutil"
 	"log"
+	"math"
```

Comment:
Yeah I agree, will just do a quick if statement since a min function would
only be used here anyway.

On Tue, Oct 20, 2015 at 6:50 PM bifurcation <notifications@github.com>
wrote:

> In cmd/shell.go
> <https://github.com/letsencrypt/boulder/pull/1000#discussion_r42576000>:
>
> > @@ -28,6 +28,7 @@ import (
> >  	"fmt"
> >  	"io/ioutil"
> >  	"log"
> > +	"math"
>
> Honestly, rather than pulling in this whole dependency just for math.Min()
> -- which doesn't even solve your problem without a lot of casting! -- I
> would prefer that you just define your own "min" function or just use an
> if statement in context.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/letsencrypt/boulder/pull/1000/files#r42576000>.
>